spec/anchor.spec.js

```diff
@@ -329,6 +329,42 @@ describe('Anchor Button TestCase', function () {
             expect(link).not.toBeNull();
             expect(link.href).toBe(validUrl.toLowerCase());
         });
+        it('should change spaces to %20 for a valid url if linkValidation options is set to true', function () {
+            var editor = this.newMediumEditor('.editor', {
+                anchor: {
+                    linkValidation: true
```

Comment:
@nmielnik updated the documentation and changed the space replacement with `encodeURI`